Renames bin.brillo to usr.bin.brillo
Installs usr.bin.brillo to /etc/apparmor.d/local
Modify usr.bin.brillo to include local version

diff --git a/Makefile b/Makefile
index a6909d8..5f78f37 100644
--- Makefile
+++ Makefile
@@ -14,6 +14,7 @@ MANDIR=$(DESTDIR)$(PREFIX)/share/man/man1
 PKEDIR=$(DESTDIR)$(PREFIX)/share/polkit-1/actions
 UDEVDIR=$(DESTDIR)$(PREFIX)/lib/udev/rules.d
 AADIR=$(DESTDIR)/etc/apparmor.d
+LAADIR=$(DESTDIR)/etc/apparmor.d/local
 
 SRC = \
 	src/value.c \
@@ -53,11 +54,12 @@ polkit:
 dist: $(PROG) man polkit
 
 install-dist: install dist
-	install -dZ -m 755 $(MANDIR) $(PKEDIR) $(UDEVDIR) $(AADIR)
+	install -dZ -m 755 $(MANDIR) $(PKEDIR) $(UDEVDIR) $(AADIR) $(LAADIR)
 	install -DZ -m 644 $(PROG).1 -t $(MANDIR)
 	install -DZ -m 644 contrib/$(VENDOR).$(PROG).policy -t $(PKEDIR)
 	install -DZ -m 644 contrib/90-brillo.rules -t $(UDEVDIR)
-	install -DZ -m 644 contrib/bin.brillo -t $(AADIR)
+	install -DZ -m 644 contrib/usr.bin.brillo -t $(AADIR)
+	install -DZ -m 644 contrib/local/usr.bin.brillo -t $(LAADIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PROG)
diff --git a/contrib/local/usr.bin.brillo b/contrib/local/usr.bin.brillo
new file mode 100644
index 0000000..f6eea46
--- /dev/null
+++ contrib/local/usr.bin.brillo
@@ -0,0 +1 @@
+# Site-specific additions and overrides for 'usr.bin.brillo'
diff --git a/contrib/bin.brillo b/contrib/usr.bin.brillo
similarity index 83%
rename from contrib/bin.brillo
rename to contrib/usr.bin.brillo
index 84583c6..e42dc69 100644
--- contrib/bin.brillo
+++ contrib/usr.bin.brillo
@@ -22,4 +22,7 @@ profile @{BRILLO} @{BRILLO_EXEC} {
   /sys/class/{backlight,leds}/   r,
   /sys/devices/**/brightness     rwk,
   /sys/devices/**/max_brightness r,
+
+  # Site-specific additions and overrides. See local/README for details.
+  #include <local/usr.bin.brillo>
 }

